src/main/java/com/amazon/opendistroforelasticsearch/security/configuration/PrivilegesInterceptorImpl.java

```diff
@@ -49,10 +55,17 @@
 import com.amazon.opendistroforelasticsearch.security.securityconf.DynamicConfigModel;
 import com.amazon.opendistroforelasticsearch.security.user.User;
 
+import com.google.common.collect.ImmutableMap;
+
 public class PrivilegesInterceptorImpl extends PrivilegesInterceptor {
 
     private static final String USER_TENANT = "__user__";
     private static final String EMPTY_STRING = "";
+    private static final String SUFFIX = "_1";
+    private static final Map<String, Object> KIBANA_INDEX_SETTINGS = ImmutableMap.of(
+            IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1,
+            IndexMetadata.SETTING_AUTO_EXPAND_REPLICAS, "0-1"
```

Comment:
Does the original ".kibana_1" index have a similar setting ? If not, should we use the same settings as original kibana index ?

Reply:
".kibana_1" is created with the same settings, number of shards 1 and auto expand replicas "0-1".